packages/steps/src/item.vue

```diff
@@ -58,7 +58,7 @@ interface IStepsProps {
 
 interface IStepInstance extends ComponentInternalInstance {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  ctx: any
+  setupState: any
```

Comment:
Do you mind enhancing the type definitions here? Since you already know what has been returned by the setups
BTW, this is not a mandatory request, if you do not have the time or you do not feel to do this, you can simply resolve my request here

Reply:
OK, I will add it later.

Reply:
Thanks @Ryan2128 !